x-pack/test/observability_functional/apps/observability/pages/rules_page.ts

```diff
@@ -163,6 +163,25 @@ export default ({ getService }: FtrProviderContext) => {
           return true;
         });
       });
+
+      it('should navigate to the details page when clicking on a rule in event logs tab', async () => {
+        await observability.alerts.rulesPage.clickLogsTab();
+        await observability.alerts.rulesPage.clickOnRuleInEventLogs();
+
+        await retry.waitFor(
```

Comment:
nit: Can we use `await testSubjects.existOrFail('ruleDetails');` here, then we don't need to retry? It also retries internally.

Reply:
https://github.com/elastic/kibana/pull/165115/commits/393cba8f269338c975b927e19ca38d6895d99cde